WebApp/testsGUI/scenarios/addBlock.spec.js

```diff
@@ -62,4 +62,93 @@ Scenario('[ADD02] Add new block with edited attributes.', async (I) => {
             type: 'lecture'
         }
     });
+});
+
+Scenario('[ADD03] Check, if subject is founded without accent', async (I) => {
+    I.click({ react: 'Block', props: { i: 3, j: 3 }});
+    I.fillField('courseName', 'architektury informacnych systemov');
+    I.pressKey("ArrowDown");
+    I.pressKey("Enter");
+    I.seeInField('courseName', 'architektúry informačných systémov (5IS208)');
+});
+
+Scenario('[ADD04] Check, if subject is founded with accent', async (I) => {
+    I.click({ react: 'Block', props: { i: 3, j: 3 }});
+    I.fillField('courseName', 'číslicové počítače');
+    I.pressKey("ArrowDown");
+    I.pressKey("Enter");
+    I.seeInField('courseName', 'číslicové počítače (5BH118)');
+});
+
+Scenario('[ADD05] Check, if founded subjects is sorted by faculty of student', async (I) => {
```

Comment:
What if some subjects get taken down ? Will it affect this ? Maybe just check Subject code which us based on faculty if it is possible. Just a thought

Reply:
Yes it will affect. Now I update commit which solve this problem.